lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js

```diff
@@ -90,6 +90,10 @@ testRule(rule, {
     {
       code: "html { --custom-property-set: {} }",
       description: "custom property set in selector"
+    },
+    {
+      code: "::-webkit-scrollbar-button:horizontal:decrement {}",
+      description: "webkit scrollbar multiple pseudo classes"
```

Comment:
Let's use `description: "webkit scrollbar chained pseudo classes"` as chained is the correct terminology I believe

Reply:
I made this change in https://github.com/stylelint/stylelint/pull/2813/commits/21cf021454fe9a07c045513c2193398117daa5ee